app/views/home/subscriptions.html.erb

```diff
@@ -46,7 +46,7 @@
         <a rel="tooltip" title="Once per day" class="btn btn-default btn-xs" href="javascript:void(0)"><i class="fa fa-list"></i> Digest</a>
         -->
         <script> input_<%= subscription.tid %> = '#share-<%= subscription.tid %>'</script>
-        <a class="btn btn-default btn-xs" rel="popover" data-placement="left" title="<%= t('home.subscriptions.share_subscription_link') %>" data-html="true" data-content="<input id='share-<%= subscription.tid %>' onClick='$(input_<%= subscription.tid %>).select()' type='text' value='https://<%= ActionController::Base.default_url_options[:host] %>/subscribe/tag/<%= subscription.tagname %>'/><div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/share' class='twitter-share-button'>Tweet</a></div> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='//platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');</script>
+        <a class="btn btn-default btn-xs" rel="popover"  data-toggle="popover" data-placement="left" title="<%= t('home.subscriptions.share_subscription_link') %>" data-html="true" data-content="<input id='share-<%= subscription.tid %>' onClick='$(input_<%= subscription.tid %>).select()' type='text' value='https://<%= ActionController::Base.default_url_options[:host] %>/subscribe/tag/<%= subscription.tagname %>'/><div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/share' class='twitter-share-button'>Tweet</a></div> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='//platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');</script>
```

Comment:
Just checking here -- is the only change adding `data-toggle="popover"`? Thanks!